src/semaphore/broadcast/markdown.py

```diff
@@ -150,10 +164,15 @@ def to_broadcast(self) -> BroadcastMessage:
         `semaphore.broadcast.data.BroadcastMessage`
             The broadcast message.
         """
+
+        new_summary, new_body = self.summarize(
+            self.metadata.summary, self.body
+        )
```

Comment:
So with the suggested change to the method, I'd look something like this:

```suggestion
        summary, body = self.extract_content()
```